app/count.js

```diff
@@ -2,6 +2,21 @@ exports = (typeof window === 'undefined') ? global : window;
 
 exports.countAnswers =  {
   count : function (start, end) {
+    var timeout;
+    function doIt () {
+      console.log(start++);
 
+      if (start <= end) {
+        timeout = setTimeout(doIt, 100);
+      }
+    }
+
+    doIt();
+
+    return {
+      cancel : function () {
+        timeout && clearTimeout(timeout);
+      }
+    };
```

Comment:
Sorry...one last comment! I think these answers should be moved over to [the answers repo](https://github.com/rmurphey/js-assessment-answers).